lib/truncate-collections/index.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+class TruncateCollection {
+
+  constructor(mongoose) {
+    this._mongoose = mongoose;
+    this.truncate = this.truncate.bind(this);
```

Comment:
it is much more important to have an easily readable code, than a one liner function :)